scripts/convert_cuml.py

```diff
@@ -21,18 +21,18 @@
 """
 
 import argparse
-import os
+import pathlib
```

Comment:
In general, let's try to leave unrelated changes out of PRs, especially version bumps PRs. Ideally, this would be the minimum changeset we need to bump to latest versions. No need to change this back, though; it's reasonably innocuous.